packages/eslint-plugin/src/rules/consistent-indexed-object-style.ts

```diff
@@ -0,0 +1,118 @@
+import { createRule } from '../util';
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+
+export default createRule({
+  name: 'consistent-indexed-object-style',
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Enforce or disallow the use of the record type',
+      category: 'Stylistic Issues',
+      // too opinionated to be recommended
+      recommended: false,
+    },
+    messages: {
+      preferRecord: 'A record is preferred over an index signature',
+      preferIndexSignature: 'An index signature is preferred over a record.',
+    },
+    fixable: 'code',
+    schema: [
+      {
+        enum: ['always', 'never'],
```

Comment:
let's make this a bit clearer.
`consistent-indext-object-style = "always"` doesn't make any sense.

Lets swap it to something like `'always-record' | 'always-index-sig'` so it's clearer what you're configuring

Reply:
I changed it to `record` and `index-signature` respectively, because I think the `always` prefix is unnecessary. Let me know if you still want me to change it.